code/modules/mob/transform_procs.dm

```diff
@@ -346,6 +346,7 @@
 	. = new /mob/living/silicon/ai(loc_landmark, null, src)
 
 	qdel(src)
+	return .

```

Comment:
`.` refers to the return value. All functions return `.` unless specifically told to return something else. This is redundant.